tee/tf/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+aws_region    = "eu-north-1"
+instance_type = "c5.xlarge"
+key_name      = ""        # or "" if you don’t need SSH
+eif_url       = "https://github.com/Cardinal-Cryptography/zkOS-monorepo/releases/download/prover-server-v0.1.0/shielder-prover-tee-837095f.eif"
```

Comment:
We can use now `https://github.com/Cardinal-Cryptography/blanksquare-monorepo/releases/download/prover-server-v0.1.0/shielder-prover-tee-837095f.eif`, I just updated release in the migrated repo

Reply:
I tested now this one and it also works